### PR TITLE
Fixed #323 - CSV widget table header misalignment.

### LINKED ISSUE #323: CSV widget table header misalignment
Upload a CSV, when viewed in the csv viewer widget, the headers are misaligned with the columns.

screen shot 2016-02-25 at 2 36 49 pm

### PR DESCRIPTION
**Pull request by essamjoubori**
This is a fix for issue #323. Make sure to update clients/web/static/built/plugin/minerva/datatables.min.css as some styles were changed.

screen shot 2016-02-26 at 2 43 48 pm

### CODE REVIEW

**Comment by mgrauer** [comment]
@essamjoubori 

Why did you need to update datatables.min.css ?

**Comment by mgrauer** [comment]
Was it a new version or what exactly?

File: web_external/js/views/widgets/CsvViewerWidget.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ minerva.views.CsvViewerWidget = minerva.View.extend({
         })).girderModal(this).on('shown.bs.modal', function () {
         }).on('hidden.bs.modal', function () {
         }).on('ready.girder.modal', _.bind(function () {
-            var table = $('table#data').dataTable({
+            $('table#data').dataTable({
                 // 'scrollCollapse': true,
```
Comment (mgrauer) [inline web_external/js/views/widgets/CsvViewerWidget.js]: Can you remove this commented line?

**Comment by essamjoubori** [comment]
@mgrauer It's not a new version. I had to change one css style for the table tr element (tbody td{white-space:nowrap} ==> tbody td{white-space:pre-wrap})
I tried to accomplish that with jQuery but it didn't work.

**Comment by mgrauer** [comment]
@essamjoubori 

Can you note that in the header of datatables.min.css?  I.e., make a note in the header of that file saying what you changed compared to the downloaded version and why.

**Pull request merged**